packages/toolpad-core/package.json

```diff
@@ -48,15 +48,13 @@
   "dependencies": {
     "@emotion/react": "11.11.4",
```

Comment:
I probably moved some of these back to `dependencies` by mistake, I guess they can be left out...
If the `@emotion` libraries are peer dependencies of `@mui/material` as they are could they be left out too?

Reply:
We shouldn't import from `@emotion/react` at all. I've removed the import in `AppProvider` as well